.rubocop_todo.yml

```diff
@@ -483,11 +452,6 @@ Style/RedundantBegin:
 Style/RedundantParentheses:
   Enabled: false
 
-# Offense count: 505
-# Cop supports --auto-correct.
-Style/RedundantSelf:
```

Comment:
I think it some cases it enhances the readability of the code. In some class method, I'd prefer to see `self.where(....)` instead of just where for example, but I don't have any particular justification for it.  I don't see how it affects refactoring though, the whole nature of redundant self is that it is (to ruby) pointless.